.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+    - "3.7"
```

Comment:
Just want to make sure you're aware variant-analytics is on python3.6. Any concern here if this is on 3.7?

Reply:
I'll update to 3.6